src/community/wps-sextante/src/main/java/org/geoserver/wps/sextante/GTDBFIterator.java

```diff
@@ -0,0 +1,53 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.wps.sextante;
+
+import java.io.IOException;
+
+import org.geotools.data.shapefile.dbf.DbaseFileReader;
+
+import es.unex.sextante.core.Sextante;
+import es.unex.sextante.dataObjects.IRecord;
+import es.unex.sextante.dataObjects.IRecordsetIterator;
+import es.unex.sextante.dataObjects.RecordImpl;
+import es.unex.sextante.exceptions.IteratorException;
+
+/**
+ * 
+ * @author Cesar Martinez Izquierdo
```

Comment:
Normally I would ask where the code came from (and if it is an open source project check the license it is made available under).

Alternatively we could reach out to the author and ask what the license is, or ask for explicit permission.  But yeah this is case where a contributor is introducing a dependency so we should check it out.

Normally we would check these things when a module has a code review (when it moves out of community).